### ISSUE TITLE
Bring back ability to pass custom props assigned to <Resource/> down

### ISSUE THREAD

**Issue by rmuradov** [marmelab/aor-permissions#28]
When I've used `<Admin />` from **aor** there were no problems with assigning some custom props to `<Resource />` tags in order to have ability to make some manipulations in component that I've specified in `menu` prop of `<Admin />` (i.e. `menu={CustomMenu}`)
Now I am using `Admin` from **aor-permissions** and those custom props are not passed down anymore

Could we apply this patch to bring back this ability?
```patch
diff --git a/src/Admin.js b/src/Admin.js
index 46c9629..563d185 100644
--- a/src/Admin.js
+++ b/src/Admin.js
@@ -56,6 +56,7 @@
         edit: await applyPermissionsToAction({ permissions, resource, action: 'edit' }),
         show: await applyPermissionsToAction({ permissions, resource, action: 'show' }),
         remove: await applyPermissionsToAction({ permissions, resource, action: 'remove' }),
+        ...resource.props
     };
 
     return newResource;
```

**Comment by djhi** [comment]
Good catch

**Comment by djhi** [comment]
Fixed in `1.1.2`
